packages/styles/src/components/datepicker.scss

```diff
@@ -98,6 +98,9 @@ span.ngb-dp-navigation-chevron {
   width: 40px !important;
   height: 40px !important;
   line-height: 40px;
+  &:focus-visible{
+    outline: forms.$input-focus-color solid forms.$input-focus-outline-thickness;
+  }
```

Comment:
With keyboard navigation, it doesn't work (check Firefox as the default color is blue, it's more obvious).